src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -83,12 +83,16 @@ const InfiniteScroll = ({
       let top;
       let height;
       let width;
+      let scrollHeight;
       if (scrollParent === document) {
         top = document.documentElement.scrollTop || document.body.scrollTop;
         height = window.innerHeight;
         width = window.innerWidth;
+        scrollHeight =
+          document.documentElement.scrolHeight || document.body.scrollHeight;
```

Comment:
Just quickly glancing to get an understanding to your approach and noticed a typo:

```suggestion
          document.documentElement.scrollHeight || document.body.scrollHeight;
```

Reply:
Thank you for pointing this out. Fixed